src/pybind/mgr/dashboard/frontend/package.json

```
@@ -33,17 +33,16 @@
     "test": "npm run test:config && jest --watch",
     "test:ci": "npm run test:config && JEST_SILENT_REPORTER_DOTS=true jest --coverage --reporters jest-silent-reporter",
     "test:config": "if [ ! -e 'src/unit-test-configuration.ts' ]; then cp 'src/unit-test-configuration.ts.sample' 'src/unit-test-configuration.ts'; fi",
-    "e2e": "npm run env_build && npm run e2e:update && ng e2e --webdriverUpdate=false",
-    "e2e:ci": "npm run env_build && npm run e2e:update && ng e2e --dev-server-target --webdriverUpdate=false",
-    "e2e:update": "npx webdriver-manager update --gecko=false --versions.chrome=$(google-chrome --version | awk '{ print $3 }')",
+    "e2e": "npm run env_build && g run ceph-dashboard:cypress-open",
```

You missed a letter ;)

```suggestion
    "e2e": "npm run env_build && ng run ceph-dashboard:cypress-open",
```

---

Fixed